PocketKit/Sources/Sync/Collection/CollectionUrlFormatter.swift

```diff
@@ -0,0 +1,47 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+
+/// A formatter for collection URLs
+public struct CollectionUrlFormatter {
```

Comment:
We could also make the functions on this type `static` so that we don't have to initialize an instance every time; there's no real "state" to this type.

Reply:
okey